monitors/icat_monitor.py

```diff
@@ -0,0 +1,107 @@
+"""
+Monitor ICAT for the latest run on an instrument. If end of run monitor is out of sync
```

Comment:
`If end of run monitor is out of sync then restart it.` Is a misleading comment in this file as that will be done elsewhere